app/assets/stylesheets/custom.scss

```diff
@@ -414,3 +449,255 @@ header {
     }
   }
 }
+
+input:focus {
+  outline: 0;
+}
+
+body {
+  background: $gray;
+}
+
+// 11. Tables (layout.scss)
+// ----------
+
+table {
+
+  tr {
+
+    &:nth-child(odd) {
+      background: #f9f9f9;
+    }
+
+    &:nth-child(even) {
+      background: #fff4f4;
+    }
+  }
+}
+
+// 02. Sidebar (admin.scss)
+// -----------
+
+.admin-sidebar {
+  background: $gray;
+
+  ul {
+    [class^="icon-"] {
+      color: $black;
+    }
+  }
+
+  li {
+    ul {
+      border-left: 1px solid $sidebar-hover;
```

Comment:
This border is almost invisible :thinking:.

Reply:
Ok, I'll delete this rule